Tests/BuildTests/BuildOperationTests.swift

```diff
@@ -14,34 +14,60 @@
 @testable import PackageModel
 
 import Basics
+import LLBuildManifest
+@_spi(DontAdoptOutsideOfSwiftPMExposedForBenchmarksAndTestsOnly)
+import PackageGraph
+import SPMBuildCore
 import SPMTestSupport
 import XCTest
 
 import class TSCBasic.BufferedOutputByteStream
 import class TSCBasic.InMemoryFileSystem
 
+private func mockBuildOperation(
+    buildParameters: BuildParameters,
```

Comment:
Could you please split the parameters here into tools and destination? I am adding a destination to the parameters in another PR related to cross-compilation which makes this separation functional.

Reply:
Ok, make sense to me